ocaml/xapi/pkg_mgr.mli

```diff
@@ -0,0 +1,62 @@
+val repoquery_sep : string
+
+(* Type of package manager supported *)
```

Comment:
The comments used in this mli are normal comments, please use documentation comment instead, `(** This is a doc comment *)`